harvesting/releases.py

```diff
@@ -163,7 +163,7 @@ def fetch_zenodo_data_versioned_dois(self):
             'Authorization': 'Bearer ' + os.environ.get('ZENODO_ACCESS_TOKEN')
         }
         r = requests.get(url, headers=headers)
-        if int(r.headers.get('x-ratelimit-remaining')) < 10:
+        if int(r.headers.get('x-ratelimit-remaining', -1)) < 10:
             # throttle requests
             time.sleep(60)
         r.raise_for_status()
```

Comment:
Would be nice to get rid of this code duplication (see lines 152-158). We can also do that in another PR later and leave this for now. Considering the sprint is ending this would make sense.

Reply:
fixed by p[ulling in some code from master